cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/SchemaIndexTest.scala

```diff
@@ -317,10 +320,10 @@ class SchemaIndexTest extends DocumentingTestBase with QueryStatisticsTestSuppor
     )
   }
 
-  @Test def use_index() {
+  @Test def use_btree_index() {
     profileQuery(
-      title = "Node index example",
-      text = "In the example below, the query uses a `Person(firstname)` node index, if it exists. ",
+      title = "Node BTREE index example",
+      text = "In the example below, a `Person(firstname)` `BTREE` node index is available.",
```

Comment:
Here you have `BTREE node index`, but then you have `relationship BTREE index` in the relationship example (same for TEXT), maybe be consistent?